sdcm/cluster.py

```diff
@@ -929,15 +976,24 @@ def start_alert_manager_thread(self):
     def silence_alert(self, alert_name, duration=None, start=None, end=None):
         return AlertSilencer(self._alert_manager, alert_name, duration, start, end)
 
+    def _dc_info_str(self):
+        dc_info = []
```

Comment:
Still missing the part reading the SCT configuration, and not printing for single AZ or single region

Reply:
It is not final version, I changed it